### ISSUE TITLE
Flywheel instancing incompatible with Parallax Occlusion Mapping

### ISSUE THREAD

**Issue by gittibene** [leon-o/iris-flw-compat#97]
Instanced blocks and block on contraptions don't render properly if POM is enabled. Is there any way to make this work?

**Comment by LuiCat** [comment]
POM requires cotangents and flywheel was not providing them. [This recent change](https://github.com/Jozufozu/Flywheel/pull/231) of flywheel might help.

**Comment by DedMaxim** [comment]
This doesn't seem to be fixed, even though the PM was merged March 6. Create contraptions are semi-transparent with just stripes of the correct texture going over them is POM is turned on. And not just Create stuff, regular tile entities like chests are also affected by this bug.
Bug only appears when "Iris & Oculus Flywheel Compat" is installed.

![2024-05-25_01 38 53](https://github.com/leon-o/iris-flw-compat/assets/41780875/cea00cde-6491-441b-aa3a-c50afb9e20bb)
![2024-05-25_01 39 08](https://github.com/leon-o/iris-flw-compat/assets/41780875/7a589fef-18a6-45bc-b257-5f80a9811253)
![2024-05-25_01 39 14](https://github.com/leon-o/iris-flw-compat/assets/41780875/01bed056-0737-49fe-a991-dd554ef5e8f0)
![2024-05-25_01 56 41](https://github.com/leon-o/iris-flw-compat/assets/41780875/f4cd42d0-8d50-472d-a3b5-0919e729e8a6)



---Environment---

Intel Xeon E5-2680v2
Nvidia GTX 1070
Samsung DDR3 ECC REG 4x8Gb
Windows 10 22H2
Nvidia Driver 551.61

Minecraft 1.20.1
Forge 47.2.20
Create 0.5.1.f
Embeddium 0.3.19
Iris & Oculus Flywheel Compat 0.2.5
Oculus 1.7.0

Tried with Complementary Reimagined 5.1.1, Bliss 2.0.4 and BSL 8.2.08. While Bliss creates stripes, as I described above, BSL leaves holes in some components like the water wheel and stripes on chests, and Complementary only leaves holes on some components without any stripes.
PBR resource pack AVPBR R3
